RULES/cc/cc.go

```diff
@@ -438,10 +481,32 @@ func (bin Binary) build(ctx core.Context) {
 	}
 
 	libsToLink := []string{}
+	libsToAlwaysLink := []string{}
 
 	for _, dep := range deps {
 		ins = append(ins, dep.Out)
 		if dep.AlwaysLink {
+			libsToAlwaysLink = append(libsToLink, fmt.Sprintf("%q", dep.Out))
+		} else {
+			libsToLink = append(libsToLink, fmt.Sprintf("%q", dep.Out))
+		}
+	}
+
+	switch toolchain.Flavor() {
+	case Windows:
```

Comment:
is there a reason you have always-link libs last on windows, but first on linux?